app/models/reporting/workload_report.rb

```diff
@@ -37,23 +41,23 @@ def received_applications_by_age
     # 0 days
     # 1 day
     # 2 days
-    # between 3(#number_of_days) and (#last_row_limit_in_days)
+    # between 3(#number_of_rows) and (#last_row_limit_in_days)
```

Comment:
I still think there's something wrong here 😂 - maybe the 3 shouldn't be there. I'm being pedantic though!